controllers/yawol-controller/loadbalancerset/loadbalancerset_controller.go

```diff
@@ -245,11 +245,16 @@ func findFirstMachineForDeletion(machines []yawolv1beta1.LoadBalancerMachine) (y
 // True if created before 10 minutes and no condition added yet
 // True if LastHeartbeatTime is > 5 minutes
 // True if a condition is not good for 5 minutes
-func shouldMachineBeDeleted(machine yawolv1beta1.LoadBalancerMachine) (bool, error) {
+func shouldMachineBeDeleted(machine *yawolv1beta1.LoadBalancerMachine) (bool, error) {
```

Comment:
The error return value is never used as an error. Can we return a plain string here?